extensions/wikia/AdEngine/js/template/roadblock.js

```diff
@@ -0,0 +1,91 @@
+/*global define, require*/
+define('ext.wikia.adEngine.template.roadblock', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.uapContext',
+	'ext.wikia.adEngine.provider.gpt.googleSlots',
+	'ext.wikia.adEngine.slot.service.slotRegistry',
+	'ext.wikia.adEngine.provider.gpt.helper',
+	require.optional('ext.wikia.adEngine.template.skin'),
+	'ext.wikia.adEngine.slotTweaker',
+	'wikia.document',
+	'wikia.window',
+	'wikia.log'
+], function (
+	adContext,
+	uapContext,
+	googleSlots,
+	slotRegistry,
+	helper,
+	skinTemplate,
+	slotTweaker,
+	doc,
+	win,
+	log
+) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.roadblock',
+		uapType = 'ruap',
+		tlbSlotEl = doc.getElementById('TOP_LEADERBOARD') || doc.getElementById('MOBILE_TOP_LEADERBOARD'),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'tlbSlotEl' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "tlbSlotEl" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)